src/test/java/edu/ucsb/cs156/example/web/ITOauth.java

```diff
@@ -42,18 +45,18 @@ public static void setupWireMock() {
             .port(8090)
             .extensions(new ResponseTemplateTransformer(true)));
 
-        WiremockServiceImpl.setupOauthMocks(wireMockServer);
+        WiremockServiceImpl.setupOauthMocks(wireMockServer, false);
 
         wireMockServer.start();
     }
     
     @BeforeEach
     public void setup() {
         // Launch playwright browser headless
-        browser = Playwright.create().chromium().launch();
+        // browser = Playwright.create().chromium().launch();
 
         // Launch playwright browser with visual
-        // browser = Playwright.create().chromium().launch(new BrowserType.LaunchOptions().setHeadless(false));
+        browser = Playwright.create().chromium().launch(new BrowserType.LaunchOptions().setHeadless(false));
```

Comment:
I wonder if we could have an environment variable for this?  

`HEADLESS=true` would be the default, and you could override with 

```
INTEGRATION=true HEADLESS=false mvn failsafe:integration-test
```

I really would prefer not to have commented out code as the way to switch between in the main branch.. fine for development, but we should explore alternatives before merging this.